ui/admin/app/routes/scopes/scope/targets/target/brokered-credential-sources.js

```diff
@@ -52,8 +52,8 @@ export default class ScopesScopeTargetsTargetCredentialSourcesRoute extends Rout
   @confirm('questions.remove-confirm')
   @notifyError(({ message }) => message, { catch: true })
   @notifySuccess('notifications.remove-success')
-  async removeCredentialSource(target, credentialLibrary) {
-    await target.removeCredentialSource(credentialLibrary.id);
+  async removeBrokeredCredentialSource(target, credentialLibrary) {
+    await target.removeBrokeredCredentialSource(credentialLibrary.id);
```

Comment:
I thought `credentialLibrary` was renamed to `credentialSource` here. Can we rename it here please?
```suggestion
  async removeBrokeredCredentialSource(target, credentialSource) {
    await target.removeBrokeredCredentialSource(credentialSource.id);
```